### ISSUE TITLE
Set language attribute

### ISSUE THREAD

**Issue by benbarnett** [moneyadviceservice/made_with_dough#8]
Fails cursory accessibility test from Tenon:

http://tenon.io/testNow.php?url=http%3A%2F%2Fwww.madewithdough.org.uk%2F
